Gutenberg/version.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# This file isolates the definition of which version of Gutenberg to use.
+# This way, it can be accessed by multiple sources withou duplication.
+
+# Either use commit or tag, if both are left uncommented, tag will take precedence.
+#
+# If you want to use a local version, please use the LOCAL_GUTENBERG environment variable when calling CocoaPods.
+#
+# Example:
+#
+#   LOCAL_GUTENBERG=../my-gutenberg-fork bundle exec pod install
+GUTENBERG_CONFIG = {
+  # commit: '',
+  tag: 'v1.94.0'
+}
```

Comment:
@geriux @fluiddot @dcalhoun @SiobhyB and crew, does having a dedicated file where to go to to modify the Gutenberg version used works with you? Have you got a script that generates the release scenario? If so, I'll make a PR there about this.

Reply:
Hey @mokagio 👋 , thanks for extracting this logic!

> does having a dedicated file where to go to to modify the Gutenberg version used works with you?

Yeah, this will make a bit easier the process of changing the version, although currently, it's not difficult to do it in the `Podfile` file.

> Have you got a script that generates the release scenario? If so, I'll make a PR there about this.

Yes, we use [this release script](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/trunk/release_automation.sh) when cutting a new version. So, once we land these changes we should update [this part](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/d77e07311d0f2cb4b220ed27c0fcf42cc9932c88/release_automation.sh#L336-L339) that updates the Gutenberg Mobile reference.

Reply:
Done via https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/114